src/lib/firebase.js

```diff
@@ -4,15 +4,16 @@ import firebase from 'firebase/app';
 import 'firebase/firestore';
 
 // Initalize Firebase.
-const firebaseConfig = {
-  apiKey: "AIzaSyDlgVTZkjq6HT0pFm7sBiSm60EQ19D2RrI",
-  authDomain: "tcl-26-shopping-list.firebaseapp.com",
-  projectId: "tcl-26-shopping-list",
-  storageBucket: "tcl-26-shopping-list.appspot.com",
-  messagingSenderId: "368940430821",
-  appId: "1:368940430821:web:5347ad9d833f3a84cd6b0c"
-};
+if (!firebase.apps.length) {
+  firebase.initializeApp({
+    apiKey: 'AIzaSyDlgVTZkjq6HT0pFm7sBiSm60EQ19D2RrI',
+    authDomain: 'tcl-26-shopping-list.firebaseapp.com',
+    projectId: 'tcl-26-shopping-list',
+    storageBucket: 'tcl-26-shopping-list.appspot.com',
+    messagingSenderId: '368940430821',
+    appId: '1:368940430821:web:5347ad9d833f3a84cd6b0c',
+  });
+}
```

Comment:
I realized the notes in the issue are misleading because the video made by Alejandro is using NextJS and it needs `if (!firebase.apps.length)` but NextJS and CRA have very different opinions on package configurations. The `if` conditional is not necessary for CRA.  Defining this way would be a clear connection between the configuration and what we're consuming down the road. 
```suggestion
const firebaseConfig = {
  apiKey: "AIzaSyDlgVTZkjq6HT0pFm7sBiSm60EQ19D2RrI",
  authDomain: "tcl-26-shopping-list.firebaseapp.com",
  projectId: "tcl-26-shopping-list",
  storageBucket: "tcl-26-shopping-list.appspot.com",
  messagingSenderId: "368940430821",
  appId: "1:368940430821:web:5347ad9d833f3a84cd6b0c"
};

const firebaseInstance = firebase.initializeApp(firebaseConfig);
```

Reply:
Thanks for clarifying, we were a bit confused about whether that check was necessary!